src/Test/Utilities/Portable/Compilation/RuntimeUtilities.cs

```diff
@@ -15,6 +15,18 @@ namespace Microsoft.CodeAnalysis.Test.Utilities
     /// </summary>
     public static partial class RuntimeUtilities
     {
+        internal static bool IsDesktopRuntime =>
+#if NET472
+            true;
+#elif NETCOREAPP2_1
+            false;
+#elif NETSTANDARD2_0
```

Comment:
why throw at runtime here instead of `#error`? Does this project get compiled against netstandard?

Reply:
The test utilities library has three target frameworks to support the following scenarios:

- net472: for unit tests libraries that target net472
- netcoreapp2.1: for unit test libraries that target netcoreapp2.1
- netstandard2.0: for all the helper libraries that are in between the test utilities and the unit test libraries.

Essentially the netstandard2.0 target framework here is just providing an API facade. The implementation is never actually used. It only exists so all of the other libraries can single target netstandard2.0.